File: tests/cases/lock/locked/csharp/unlocked/inputs/baseline/Orders/Billing.cs
using CodeConstraints.Rules;

namespace Orders;

public class Receipt
{
    public decimal Total { get; }
    public int Lines { get; }

    public Receipt(decimal total, int lines) { Total = total; Lines = lines; }

    [Locked(Reason = "receipt wording is contractual; finance signed off on it")]
    public string Formatted() { return $"{Lines} line(s) - total {Total}"; }
}

File: tests/cases/lock/locked/python/changed/inputs/src/orders/billing.py
from cdec_rules import locked


class Receipt:
    def __init__(self, total: float, lines: int) -> None:
        self.total: float = total
        self.lines: int = lines

    @locked(reason="receipt wording is contractual; finance signed off on it")
    def formatted(self) -> str:
        return f"{self.lines} item(s) - total {self.total:.2f}"
